bundles/mapping/mapmodule/oskariStyle/constants.js

```diff
@@ -45,12 +45,12 @@ export const EFFECT = {
 };
 
 export const FILL_STYLE = {
-    THIN_DIAGONAL: 0,
-    THICK_DIAGONAL: 1,
-    THIN_HORIZONTAL: 2,
-    THICK_HORIZONTAL: 3,
-    TRANSPAREN: 4,
-    SOLID: 5
+    THIN_DIAGONAL: 'thin_diagonal',
+    THICK_DIAGONAL: 'thick_diagonal',
```

Comment:
The change of these values creates a breaking change for current styles used over RPC API, the printing backend implementation and the current styles in the database. Changing these should be in another PR AND would require fallback for the current values that are supported.

Also I would rather have the value as diagonal/horizontal first (as the "main" style direction) and thin/thick after it (as a modifier for the style). But I'm open for discussing this in more details. The actual problem is that "everything breaks" for most styles using these if we make just this change without adding support for the current values also (now only the style editor supports these values, not the actual components that end-users see with these styles).

Reply:
Fixed these back to old one.